met-web/src/components/layout/Header.tsx

```diff
@@ -1,36 +1,119 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import AppBar from "@mui/material/AppBar";
 import Toolbar from "@mui/material/Toolbar";
 import Typography from "@mui/material/Typography";
 import Container from "@mui/material/Container";
 import { useAppSelector } from "../../hooks";
-import sx from "mui-sx";
+import UserService from "../../services/UserServices";
+import Button from "@mui/material/Button";
+import styled from "@emotion/styled";
 
+import sx from "mui-sx";
+// import BCGOVLOGO from '../../assets/logo.png';
 const Header = () => {
+  useEffect(() => {
+    setIsLoggedIn(UserService.isLoggedIn);
+  });
+
   const authenticated = useAppSelector(
     (state) => state.user.authentication.authenticated
   );
 
+  const [isLoggedIn, setIsLoggedIn] = useState<boolean>(false);
+
   const sxLight = {
     color: "#494949",
-    backgroundColor: "#F7F8FA",
+    backgroundColor: "#f2f2f2",
+    boxShadow: "0px 5px 9px #29",
   };
 
+  const LogoContainer = styled.div`
+    display: flex;
+    flex-direction: row;
+    flex: 1.3;
+    align-items: flex-end;
+    justify-content: flex-end;
+  `;
+
+  const TitleContainer = styled.div`
+    display: flex;
+    flex-direction: row;
+    flex: 3;
+    align-items: flex-end;
+    justify-content: flex-start;
+  `;
+
+  const LogoutContainer = styled.div`
+    display: flex;
+    flex: 1;
+    align-items: center;
+    justify-content: center;
+  `;
+
```

Comment:
It seems that every component in the header has its own flex container and its own flex alignment. Are we not able to have them under the same alignment rules in one flex container? Also as of now they are a bit misaligned